tests/neg/i22440.check

```diff
@@ -3,5 +3,5 @@
   |            ^
   |            Implicit parameters should be provided with a `using` clause.
   |            This code can be rewritten automatically under -rewrite -source 3.7-migration.
-  |            To disable the warning, please use the following option: 
+  |            To disable the warning, please use the following option:
```

Comment:
it's nice to find a like-minded dev. By coincidence, today I:
![image](https://github.com/user-attachments/assets/3b614aa5-2f52-40c5-8bce-cbf54773e764)

Reply:
My editor removes the space and I couldn't be bother to find the setting responsible for it :S